machine-runner/src/runner/runner.ts

```diff
@@ -364,16 +443,23 @@ namespace MachineRunnerIterableIterator {
 export type NextValueAwaiter = ReturnType<typeof NextValueAwaiter['make']>
 
 namespace NextValueAwaiter {
-  export const make = ({
+  export const make = <
+    SwarmProtocolName extends string,
+    MachineName extends string,
+    RegisteredEventsFactoriesTuple extends MachineEvent.Factory.Any[],
```

Comment:
this seems unused

Reply:
It isn't, Fixing this. How did you catch this? Anyway, fixed in https://github.com/Actyx/machines/pull/29/commits/85ed0b3f30efb6990caf97ab1e6de1b826f78653